_posts/2022-04-24-sr2022-knockouts.md

```diff
@@ -0,0 +1,117 @@
+---
+title: TEAM 1st wins Student Robotics Competition 2022!
+image: image
+image_alt: Alt text for the image
+description: SR2022 came to an exciting conclusion with a victory for {TEAM 1st}
+---
+
+That’s it for Student Robotics 2022! {TEAM 1st} took the crown in an intense
+showdown between {TEAM 2}, {TEAM 3}, and {TEAM 4}. It was fantastic to finally
+see robots in-person again after two years of being virtual. The ingenuity and
+engineering displayed by all our teams was incredible and we're delighted with
+the performances we saw over the weekend.
+
+## The Game: This Way Up
+
+This Way Up challenged teams to collect the most tin cans from the arena and
+ensure that they were the right way up for maximum points. When in a team’s
+scoring zone, correctly oriented cans were worth 3 points, upside down cans were
+worth one point, and cans lying on their side were worth nothing. However, not
+all cans were created equal: tin cans on the arena floor started upside down,
+but those on the raised platform in the centre were already the right way up,
+making them harder to get but easier to score points with. Robots could detect a
+can’s orientation by the coloured insulating band along the bottom. The states
+of the cans were recorded at the end of the matches and the teams were scored
+accordingly. {INSERT PICTURE OF ARENA}
```

Comment:
Presumably this could use the diagram from the rules? Or do we want a photo of the competition arena?

Reply:
Yeah diagram or picture of the arena would both work. I thought picture of the actual arena might be slightly nicer, particularly as it would also show the raised platform more clearly.